spec/lib/tito_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "climate_control"
+require "json"
+require "webmock/rspec"
+require "tito"
+
+describe Tito do
+  around do |example|
+    ClimateControl.modify TITO_AUTH_TOKEN: 'bar' do
+      example.run
+    end
+  end
+
+  describe "#event_list" do
+    it "returns a list of events" do
+      event_data = { "title" => "event info" }
```

Comment:
I'm not sure I'm understanding what you're saying. Are you suggesting that we use the same method for both tests or that all this setup should be put under the `stub_tito_request_with` method (and maybe rename it as a successful one)?